templates/user/notification/notification_div.tmpl

```diff
@@ -59,24 +59,24 @@
 									{{end}}
 								</td>
 								<td class="eleven wide">
-									<a class="item" href="{{.Link}}">
+									<a class="item issue-title muted" href="{{.Link}}">
 										{{if $issue}}
-											#{{$issue.Index}} - {{$issue.Title}}
+											#{{$issue.Index}} - {{$issue.Title | RenderEmoji $.Context | RenderCodeBlock}}
 										{{else}}
 											{{$repo.FullName}}
 										{{end}}
 									</a>
 								</td>
 								<td>
-									<a class="item" href="{{$repo.Link}}">{{$repo.FullName}}</a>
+									<a class="item muted" href="{{$repo.Link}}">{{$repo.FullName}}</a>
 								</td>
 								<td class="collapsing">
 									{{if ne .Status 3}}
 										<form action="{{AppSubUrl}}/notifications/status" method="POST">
 											{{$.CsrfTokenHtml}}
 											<input type="hidden" name="notification_id" value="{{.ID}}">
 											<input type="hidden" name="status" value="pinned">
-											<button class="ui mini button" title='{{$.locale.Tr "notification.pin"}}'
+											<button class="ui mini button button-link" title='{{$.locale.Tr "notification.pin"}}'
```

Comment:
Is it a `link`?

This is a button, it does post, it doesn't nagivate like a link.

So maybe it could be called "simple button" or "plain button"?

Reply:
And this button looks really strange, it is in a form, which has action url, but itself also has `data-url` and `data-status` attributes .... 

Really curious how it works (I haven't looked into details, too late, but I guess there is something not ideal ....)

Reply:
I think `button-link` perfectly describes what it does.

Reply:
Maybe we can find some more general agreements. How do other mainstream framekworks call such button?

Reply:
Boostrap calls it `btn-link`
Primer calls it `Button--link`

That alone should be enough for you to accept it.

Reply:
https://primer.style/css/components/buttons#link-button

It seems that primer also calls it `btn-link`, the same as bootstrap? Should we also use `btn-link` 😄 

Then I guess we can drop `.ui.tertiary.button` in the future?

Reply:
But, one more thing: https://primer.style/css/components/buttons#link-button : it has `underline` style .....

Reply:
> Should we also use btn-link

That would be inconsistent with Fomantic

> They all have underline style .....

Only bootstrap has and we do not render links with underline anywhere unless hovered. I find it unsuitable and outdated.

Reply:
Thank you for the detailed explanation. Very clear.